test/unit/update-sync.test.js

```diff
@@ -109,4 +109,39 @@ describe('etch.updateSync(component)', () => {
 
     expect(events).to.eql(['child-write', 'parent-write', 'child-read', 'parent-read'])
   })
+
+  it('throws a generic exception if undefined is returned from render', () => {
```

Comment:
Think it's worth mentioning that the generic exception is thrown *when there is no class*. It confused me for a sec how this differed from the test below.